dbt/models/default/docs.md

```diff
@@ -31,7 +31,6 @@ View that standardizes residential property characteristics for use in modeling
 and reporting.
 
 ### Assumptions
-`char_renovation` (`dweldat.user3`) is updated when HI permits are filed.
```

Comment:
**[Nitpick, non-blocking]** Seems like this doc no longer has any assumptions, should we remove the stub `### Assumptions` header then?

Reply:
Nay, I'm building it out in a separate PR so this will change quite soon anyways.